lib/company/opening.js

```diff
@@ -348,10 +348,44 @@ function getAllOpeningsOfCompany(userId, callback) {
   );
 }
 
+/**
+ * Function to get details of opening by opening_id
+ * @param {ObjectId} openingId opening table object Id
+ * @param {ObjectId} userId user Id
+ * @param {Function} callback  having error and data
+ */
+function getOpeningDetails(openingId, userId, callback) {
+  JobOpening.find({
+    _id: mongoose.Types.ObjectId(openingId),
+  })
+    .populate({
```

Comment:
Run this function and add a sample response . This might will not give all the data as per required by the API.

Reply:
added functionality as per told